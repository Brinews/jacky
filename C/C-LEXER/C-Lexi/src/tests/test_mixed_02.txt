
9999.00000000001x08
9999.00000000001 x08

123x08123456.4
123x09123456.7

IFIF WHILEIF DOTHEN PRINTIF {}
