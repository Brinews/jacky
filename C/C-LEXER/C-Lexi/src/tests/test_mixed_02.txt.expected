{9999.00000000001 , REALNUM , 2}
{x08 , ID , 2}
{9999.00000000001 , REALNUM , 3}
{x08 , ID , 3}
{123x08 , BASE08NUM , 5}
{123456.4 , REALNUM , 5}
{123 , NUM , 6}
{x09123456 , ID , 6}
{ , DOT , 6}
{7 , NUM , 6}
{IFIF , ID , 8}
{WHILEIF , ID , 8}
{DOTHEN , ID , 8}
{PRINTIF , ID , 8}
{ , ERROR , 8}
{ , ERROR , 8}
{ , END_OF_FILE , 10}
